modules/gapi/test/render/gapi_render_tests_ocv.cpp

```diff
@@ -493,15 +493,41 @@ INSTANTIATE_TEST_CASE_P(RenderNV12OCVTestFTextsImpl, RenderNV12OCVTestFTexts,
                             Values(cv::Scalar(0, 255, 0))));
 #endif // HAVE_FREETYPE
 
+// FIXME Implement a macros to instantiate the tests because BGR and NV12 have the same parameters
+
 INSTANTIATE_TEST_CASE_P(RenderBGROCVTestMosaicsImpl, RenderBGROCVTestMosaics,
                         Combine(Values(cv::Size(1280, 720)),
```

Comment:
@dmatveev Consider avoiding using of HD-sized images in accuracy tests.

These tests are filtered from:
- Debug pre-commit build
- coverage
- valgrind

Only VGA and less sized images are run for these builder types.

---

BTW, If it is highly required to work with HD resolution, then don't make them parametrized.